pipeline/setup-runner.py

```diff
@@ -0,0 +1,347 @@
+                "/InstallMenus=No",
+            ],
+            capture_output=True,
+            text=True,
+            check=False,
+        )
+
+        print(f"Installation exit code: {result.returncode}")
+        if result.stdout:
+            print(f"Installation output: {result.stdout}")
+        if result.stderr:
+            print(f"Installation errors: {result.stderr}")
+
+        # Verify installation
+        print(f"Checking installation at {houdini_dir}")
+        if houdini_dir.exists():
+            print("Directory exists, contents:")
+            run(["cmd", "/c", "dir", str(houdini_dir)], check=False)
+            hython_exe = houdini_dir / "bin" / "hython.exe"
+            if hython_exe.exists():
+                print(f"SUCCESS: hython.exe found at {hython_exe}")
+                houdini_marker.touch()  # Create marker on success
+            else:
+                print(f"ERROR: hython.exe NOT found at {hython_exe}")
+        else:
+            print(f"ERROR: Installation directory {houdini_dir} does not exist")
+
+        houdini_installer.unlink(missing_ok=True)
+
+    print("Installing Houdini submitter...")
+    for version in HOUDINI_VERSIONS:
+        major_minor = ".".join(version.split(".")[:2])
+        print(f"Installing submitter for Houdini {version} (major_minor: {major_minor})")
+        result = subprocess.run(
+            ["hatch", "run", "install", "--houdini-version", major_minor],
+            capture_output=True,
+            text=True,
+            check=False,
+        )
+        print(f"Hatch install exit code: {result.returncode}")
+        if result.stdout:
+            print(f"Hatch install stdout: {result.stdout}")
+        if result.stderr:
+            print(f"Hatch install stderr: {result.stderr}")
+        if result.returncode != 0:
+            print(f"ERROR: Hatch install failed for Houdini {version}")
+            sys.exit(result.returncode)
+
+        # Install pywin32 into Houdini's Python
+        houdini_dir = Path(f"C:/Tools/houdini-{version}")
+
+        # Map Houdini versions to Python versions
+        if version.startswith("19.5"):
+            python_version = "3.9"
+        elif version.startswith("20.0"):
+            python_version = "3.10"
+        elif version.startswith(("20.5", "21.0")):
+            python_version = "3.11"
+        else:
+            python_version = "3.11"
+
+        site_packages = (
+            houdini_dir / f"python{python_version.replace('.', '')}" / "lib" / "site-packages"
+        )
+
+        print(f"Installing pywin32 into Houdini {version} Python (version {python_version})...")
+        run(
+            [
+                "pip",
+                "install",
+                "pywin32<311,>=307",
+                f"--python-version={python_version}",
+                "--only-binary=:all:",
+                f"--target={site_packages}",
+                "--no-cache-dir",
+                "--force-reinstall",
+            ]
+        )
```

Comment:
Same as my other feedback. Is there any way to make this dynamic and not have to be maintained in two places?